go/vt/vtgate/planbuilder/testdata/union_cases.txt

```diff
@@ -1808,11 +1855,15 @@ Gen4 plan same as above
   "Instructions": {
     "OperatorType": "SimpleProjection",
     "Columns": [
-      1
+      2
     ],
     "Inputs": [
       {
         "OperatorType": "Distinct",
+        "Collations": [
+          "(0:1)"
+        ],
+        "ResultColumns": 1,
```

Comment:
This is wrong. SimpleProjection on top of this needs to use the literal 1 that has been pushed down here

Reply:
this is resolved via 9fd0332